src/cairo_types/bigint.rs

```diff
@@ -0,0 +1,9 @@
+use cairo_type_derive::FieldOffsetGetters;
+use cairo_vm::Felt252;
+
+#[derive(FieldOffsetGetters)]
+pub struct BigInt3 {
+    pub d0: Felt252,
+    pub d1: Felt252,
+    pub d2: Felt252,
+}
```

Comment:
I avoided adding this in #166 just to reduce code, but I didn't have a strong conviction about doing so... If we're adding it here anyway, that hint could be updated to use it.

Reply:
Up to you!